daemon/control.c

```diff
@@ -1000,8 +1000,9 @@ control_handle_message(control_t *control, const ControllerToDaemon *msg, int fd
 	} break;
 
 	case CONTROLLER_TO_DAEMON__COMMAND__REBOOT_DEVICE: {
-		cmld_reboot_device();
```

Comment:
seems you implicitly reverted some changes from lately merged PRs.

Reply:
Done